velero-2-mixin/panels.libsonnet

```diff
@@ -0,0 +1,97 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local utils = commonlib.utils;
+{
+  new(this):
+    {
+      local t = this.grafana.targets,
+      local stat = g.panel.stat,
+      local alertList = g.panel.alertList,
+
+      // create stat panel using commonlib
+      successfulBackupsCount:
+        commonlib.panels.generic.stat.info.new(
+          'Successful backups',
+          targets=[t.succesfulBackups],
+          description='Number of successful backups'
+        )
+        + stat.options.withGraphMode('area'),
+
+      failedBackups:
+        commonlib.panels.generic.stat.info.new(
+          'Failed backups',
+          targets=[t.failedBackups],
+          description='Number of failed backups.'
+        )
+        + stat.options.withGraphMode('area'),
+
+      successfulRestores:
+        commonlib.panels.generic.stat.info.new(
+          'Succesful restores',
+          targets=[t.succesfulRestores],
+          description='Number of succesful restores.'
+        )
+        + stat.options.withGraphMode('area'),
```

Comment:
Do we normally do `area` now for the stat option? Are there any scaling concerns with this size when multiple clusters are added in?

Reply:
I added it in here since these stat panels use increases. I do not believe there should be any scaling concerns since they are just summed across clusters